aws/resource_aws_codebuild_project.go

```diff
@@ -212,6 +212,7 @@ func resourceAwsCodeBuildProject() *schema.Resource {
 								codebuild.EnvironmentTypeLinuxContainer,
 								codebuild.EnvironmentTypeLinuxGpuContainer,
 								codebuild.EnvironmentTypeWindowsContainer,
+								codebuild.EnvironmentTypeArmContainer,
```

Comment:
Can you please add this new value to the [resource documentation](https://www.terraform.io/docs/providers/aws/r/codebuild_project.html#type-2), e.g. `website/docs/r/codebuild_project.html.markdown` and preferably add a new test (you can copy-paste-modify the `TestAccAWSCodeBuildProject_WindowsContainer` and its associated configuration). Thanks.

Reply:
Ok, i do it